src/plugins/management/public/components/management_app/management_app.tsx

```diff
@@ -0,0 +1,95 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+import React, { useState, useEffect, useCallback } from 'react';
+import {
+  AppMountContext,
+  AppMountParameters,
+  ChromeBreadcrumb,
+  ScopedHistory,
+} from 'kibana/public';
+import { I18nProvider } from '@kbn/i18n/react';
+import { EuiPage } from '@elastic/eui';
+import { ManagementStart } from '../../types';
+import { ManagementSection, MANAGEMENT_BREADCRUMB } from '../../utils';
+
+import { ManagementRouter } from './management_router';
+import { ManagementSidebarNav } from '../management_sidebar_nav';
+import { reactRouterNavigate } from '../../../../kibana_react/public';
+
+import './_management_app.scss';
```

Comment:
When importing SASS files directly in JS files, please remove the `_` underscore from the file name.

Reply:
done